server/bin/fxa-content-server.js

```diff
@@ -31,6 +31,16 @@ const serveStatic = require('serve-static');
 const config = require('../lib/configuration');
 const raven = require('../lib/raven');
 
+const userAgent = require('../lib/user-agent');
+if (! userAgent.isToVersionStringSupported()) {
+  // npm@3 installs the incorrect version of node-uap, one without `toVersionString`.
+  // To ensure the correct version is installed, check toVersionString is available.
+  logger.error('dependency.version.error', {
```

Comment:
I think there might be a `logger.critical` if you want an even stronger error, but we don't use it elsewhere so we can leave that out

Reply:
Updated to `logger.critical`